### PR TITLE
Add encoding support

### PR DESCRIPTION
**Pull request by hansharhoff**
This pull request an encoding argument to the  test_dir call in test(). This is related to an issue #1306 
A separate pull request has been sent to hadley/testthat (https://github.com/hadley/testthat/pull/518)

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # devtools 1.12.0.9000
 
+* Add test of `filter` parameter of `test()`.
```
Comment (hadley) [inline NEWS.md]: I think this is now out of date

Comment (hansharhoff) [inline NEWS.md]: Agreed. Removed.

File: NEWS.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
...
+
+* Add encoding support in `test_dir()` call by adding reference to pkg$encoding (#1306, @hansharhoff)
```
Comment (hansharhoff) [inline NEWS.md]: I added a line break. The issue is #1306. Do I need more information here?

**Comment by hadley** [comment]
In the future, please include `Fixes #issue-number` in one of the commit descriptions

**Comment by krlmlr** [comment]
Fixes #1306.

**Pull request merged**